romancal/dq_init/dqflags.py

```diff
@@ -0,0 +1,116 @@
+""" Roman Data Quality Flags
```

Comment:
Is this module necessary? There's a similar one in romancal/daatmodels. The two funcitons are now in stcal and should be called from there. romancal should define only pixeldq and groupdq (in datamodels).

Reply:
I removed the two functions and imported them from stcal.

Reply:
The question is why is this module necessary? Isn't all the information already in datamodels/dqflags.py? Isn't all that's needed the pixeldq and groupdq mappings? The methods should be called from stcal when needed. Or do I misunderstand something?

Reply:
It defines the values of the pixel flags, which are not the same for roman as they are for JWST, so the flag definitions don't belong in STCAL.

Reply:
Since other steps are using them as well it may be better to put them in romancal/lib?

Reply:
Moved dqflags.py to lib.